### PR TITLE
Fix support contract deployment on node

### LINKED ISSUE #18: Wavelet-client contract deployment fails on node

### PR DESCRIPTION
**Pull request by claudiucelfilip**
- allowed use of native BigInt (if available) for `bigint-buffer` which is needed on node environment. Fixes #18 
- exposed Buffer for easier, direct usage in browsers
- the number-like transaction params, now support Number, string, JSBI.BigInt and native BigInt types
- improved ad-hoc tests by adding node contract deployment use case

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>UMD Test</title>
```
Comment (ryanswart) [inline index.html:4]: Is this file here on purpose?

File: src/payloads.js [hunk 2]
```diff
@@ -161,6 +159,12 @@ export const parseFunctionParams = (...params) => {
 };
 
 export const getTransaction = (tag, nonce, block, innerPayload) => {
+
+    // toBufferBE breaks in noced as it expects BigInt values
```
Comment (ryanswart) [inline src/payloads.js]: noced?

**Review by ryanswart — COMMENTED** [review]
Nice tests, only have minor comments. Also don't know about the index.html in the root folder, other than that looks good

File: index.html (hunk 1 above)
Comment (claudiucelfilip) [reply index.html:4]: Yes, you can run `yarn test:umd` and it'll bring up a simple http-server. 
I Initially placed it inside the test folder, however, it couldn't load the JS from one level up in the folder structure.

File: src/payloads.js (hunk 2 above)
Comment (claudiucelfilip) [reply src/payloads.js]: Fixed. Thanks.

**Review by ryanswart — APPROVED** [review]
LGTM

**Pull request merged**